### DISCUSSION TITLE
Combine Output of Two REST EndPoint Together

### DISCUSSION THREAD

**Discussion by KushagraBindal** [graphql-java/graphql-java#2362] [Q&A]
Hello Guys,

I have two REST End points taking similar input arguments to load a page. With GraphQL I want to merge the outcome of these two REST Endpoint into a single GRAPHQL API hit. Can you please suggest the way to achieve the same?

**Comment by bbakerman** [comment]
graphql-java is an engine and knows nothing about HTTP - it wont inherently combine REST calls in any way - you need to do that

You need to write a graphql implementation and write a single field with a DataFetcher that makes 2 HTTP calls in its implementation to get that info.
